stable/kapacitor/values.yaml

```diff
@@ -0,0 +1,34 @@
+## influxdb image version
+## ref: https://hub.docker.com/r/library/influxdb/tags/
+##
+image:
+  repository: "kapacitor"
+  tag: "1.1.0"
+  pullPolicy: "IfNotPresent"
+
+## Specify a service type
```

Comment:
Would suggest crunching these two lines down to one and adding the trailing line with the double hashes to match the rest.